sprint.go

```diff
@@ -23,6 +23,11 @@ type IssuesInSprintResult struct {
 	Issues []Issue `json:"issues"`
 }
 
+type GetIssuesForSprintOptions struct {
```

Comment:
Remove search options embed and implement the query.Encoder interface 

See comment: https://github.com/andygrunwald/go-jira/pull/353#issuecomment-784989306